### ISSUE TITLE
Incompatibility between Thermal Expansion and Applied Energistics 2

### ISSUE THREAD

**Issue by Lodran** [CoFH/Feedback#594]
Whenever a Redstone furnace attempts to insert a new item into an inscriber, it does something that causes the inscriber to reset its progress, preventing the inscriber from ever completing its task.

Inserting a "buffer" object (such as an XU Transfer Node) between the furnace and the inscriber eliminates the problem.

https://github.com/AppliedEnergistics/Applied-Energistics-2/issues/1563

**Comment by KingLemming** [comment]
Relevant code: https://github.com/CoFH/CoFHLib/blob/master/src/main/java/cofh/lib/util/helpers/InventoryHelper.java#L130

It works with literally every other inventory in existence, including ones which have limited stacksizes. The issue is that AE2 assumes if a certain method is called, progress gets reset, end of story. I don't believe I agree with that as it allows other mods to adversely affect their block in by calling methods that everyone else seems to be okay with.

Basically the litmus test is the vanilla furnace - does it reset progress when we put stuff in it? It doesn't. They should probably behave the same way.
